patches/Terraria/Terraria/Main.cs.patch

```diff
@@ -481,22 +481,22 @@
  			CreditsRollEvent.UpdateTime();
  			WorldGen.mysticLogsEvent.UpdateTime();
  			PylonSystem.Update();
-+			
++
 +			if (dedServ)
 +				goto SkipMoonLordEffects;
-+			
++
  			if (NPC.MoonLordCountdown > 0) {
  				float num5 = MathHelper.Clamp((float)Math.Sin((float)NPC.MoonLordCountdown / 60f * 0.5f) * 2f, 0f, 1f);
  				num5 *= 0.75f - 0.5f * ((float)NPC.MoonLordCountdown / 3600f);
-@@ -49869,6 +_,8 @@
- 			else if (Terraria.Graphics.Effects.Filters.Scene["MoonLordShake"].IsActive()) {
+@@ -49870,6 +_,8 @@
  				Terraria.Graphics.Effects.Filters.Scene.Deactivate("MoonLordShake");
  			}
-+			
-+			SkipMoonLordEffects:
  
++			SkipMoonLordEffects:
++
```

Comment:
Your modification to this patch here removed the newline above this label but added a newline below it, unsure of if that was intentional.
It also caused the differ to add an extra line of patch context.

Reply:
I don't see a problem.
Both patches result in
```cs
// Empty line
SkipMoonLordEffects:
// Empty line
```

Reply:
And the context line was just changed. There is one less at the top, but one extra at the bottom